### PR TITLE
Add the linters

### PR DESCRIPTION
**Pull request by jips-on**
Add the linters

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+</head>
+<body>
+    <div>
+    <nav class="bar">
+       <a href="#"><h2 class="logo">My Logo</h2></a>
+        <div class="icon">
+            <a href="#" alt="bars"><i class="fa-solid fa-bars"></i></a>
+        </div>
+    </nav>
+</div>
```
Comment (ha-manel) [inline index.html]: - [ ] Kindly replace this `div` tag with `header` tag for semantic reasons 👍

File: index.html [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    <div>
```
Comment (ha-manel) [inline index.html]: - [ ] Kindly replace this `div` tag with `section` tag for semantic reasons 💯

File: index.html [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+              If you like what you see and have a project you need coded, don’t hestiate to contact me.</b></p>
+        </div>
+            <br>
+                <h3>LET'S CONNECT</h3>
+            <div class="container"> 
+              <a href="#"><i class="fab fa-facebook-f"></i></a>
+              <a href="#"><i class="fa-brands fa-linkedin"></i></a>
+              <a href="#"><i class="fa fa-github" aria-hidden="true"></i></a>
+              <a href="#"><i class="fa-brands fa-twitter"></i></a>  
+            </div>
```
Comment (ha-manel) [inline index.html]: - [ ] It's more semantically correct if the social media icons were a list `ul` where each icon is a list item `li` that has an anchor tag `a` 👍 Kindly replace the div with the previously mentioned structure.
- [ ] Also, kindly try to follow the design provided in Figma for this section, the size, the color, and the spacings are much different.

## Your design:
image

## Figma design:
image

File: index.html [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+            <div class="elipse14"></div>
+            <div class="elipse8"></div>
+            <div class="elipse11"></div>
+            <div class="rectangle6"></div>
+            <div class="elipse12"></div>
+            <div class="elipse13"></div>
+            <div class="elipse4"></div>
+            <div class="rectangle5"></div>
+            <div class="elipse5"></div>
+            <div class="rectangle7"></div>
```
Comment (ha-manel) [inline index.html]: - [ ] Kindly remove the background images from HTML and put them in CSS as `background-image` of the section. There is no need to download each shape on its own, you can download the whole picture directly from Figma. Kindly check the instructions in this [video](https://www.loom.com/embed/167236d17f104fc18298c5c9888354c9).

File: index.html [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+        <title>Personal portfolio</title>
+</head>
+<body>
+    <div>
+    <nav class="bar">
+       <a href="#"><h2 class="logo">My Logo</h2></a>
+        <div class="icon">
+            <a href="#" alt="bars"><i class="fa-solid fa-bars"></i></a>
+        </div>
+    </nav>
```
Comment (ha-manel) [inline index.html]: - [ ] Kindly try to follow the design provided in Figma as much as possible, this includes `font-family`, `font-size`, `color`, `margins` and `paddings`...

## Your design:
image

## Figma design:
image

File: index.html [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+<header>
+    <h1 class="name">I’m JIPSON <br>
+        Glad to see you!</h1>
+  </header>
+        <p class="ABout"><b>I’m a software developer! I can help you build a product ,
+             feature or website Look through some of my work and experience!
+              If you like what you see and have a project you need coded, don’t hestiate to contact me.</b></p>
+        </div>
+            <br>
+                <h3>LET'S CONNECT</h3>
```
Comment (ha-manel) [inline index.html]: - [ ] Kindly try to follow the design provided in Figma as much as possible, this includes font-family, font-size, color, margins and padding...

## Your design:
image

## Figma design:
image

**Review by ha-manel — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️

## Hi @jips-on 👋 
Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍  



## Highlights:  
* All tests passed ✔️

## Required Changes:
- [ ] Kindly update the pull request's title to make it descriptive of the work that has been done here 👍 
- [ ] Kindly update the pull request's description. It should include a summary descriptive of all the changes that were made in this branch 👍 

_Kindly check the comments below_

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/7507e0edea.js" crossorigin="anonymous"></script>
+        <title>Personal portfolio</title>
+</head>
+<body>
+    <header>
+    <nav>
+       <h2>My Logo</h2>
```
Comment (Nemwel-Boniface) [inline index.html]: - [ ] [OPTIONAL] As many people often use the website logo to help navigate them to the main page of the website, I would strongly advise you to make this a link. You can do so by nesting it as follows ``` <h2><a href="#">My Logo</a></h2>```

**Review by Nemwel-Boniface — APPROVED** [review]
## Hello @jips-on         :wave: ,
I am Nemwel your reviewer for this PR :man_beard:  

Thank you for making all the requested changes good job! :partying_face: 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! :partying_face: 🎉 



## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 
Met all the milestone requirements ✔️ 

## [Optional] suggestions

- [ ] [OPTIONAL] It would be a good thing if you would make your pull request descriptive. Having it as just "Add the linters" does not help much to tell what changes you made and why you would like to merge them into your main project branch. 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**